spanner/metrics_monitoring_exporter.go

```diff
@@ -74,21 +77,22 @@ func (e errUnexpectedAggregationKind) Error() string {
 // Google Cloud Monitoring.
 // Default exporter for built-in metrics
 type monitoringExporter struct {
-	shutdown     chan struct{}
-	client       *monitoring.MetricClient
-	shutdownOnce sync.Once
-	projectID    string
+	shutdown               chan struct{}
+	client                 *monitoring.MetricClient
+	shutdownOnce           sync.Once
+	projectID, compression string
```

Comment:
nit: the formatting seems a bit strange here. Could we have `compression` on a separate line?